networkx/algorithms/tests/test_dag.py

```diff
@@ -775,3 +775,51 @@ def test_compute_v_structures():
     G = nx.DiGraph(edges)
     v_structs = set(nx.compute_v_structures(G))
     assert len(v_structs) == 2
+
+
+def test_compute_v_structures_deprecated():
+    G = nx.DiGraph()
+    with pytest.deprecated_call():
+        nx.compute_v_structures(G)
+
+
+def test_v_structures_raise():
+    G = nx.Graph()
+    pytest.raises(nx.NetworkXNotImplemented, nx.dag.v_structures, G)
```

Comment:
context manager for testing raises is preferred now. It shows the code in a more readable way.
```suggestion
    with pytest.raises(nx.NetworkXNotImplemented, match="for undirected type"):
        nx.dag.v_structures(G)
```